pkgs/desktops/gnome-3/core/gcr/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, pkgconfig, intltool, gnupg, p11-kit, glib
```

Comment:
This file has been moved to libraries.

Reply:
oops, copied this one over during the rebase. Fixed.